From: Andrew Morton <akpm@linux-foundation.org>
Subject: mm-gupc-refactor-check_and_migrate_movable_pages-fix
Date: Wed Aug 24 02:10:22 PM PDT 2022

mm/gup.c: In function 'check_and_migrate_movable_pages':
mm/gup.c:2045:13: error: unused variable 'ret' [-Werror=unused-variable]
 2045 |         int ret;
      |             ^~~

Cc: Alistair Popple <apopple@nvidia.com>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 mm/gup.c |    1 -
 1 file changed, 1 deletion(-)

--- a/mm/gup.c~mm-gupc-refactor-check_and_migrate_movable_pages-fix
+++ a/mm/gup.c
@@ -2066,7 +2066,6 @@ err:
 static long check_and_migrate_movable_pages(unsigned long nr_pages,
 					    struct page **pages)
 {
-	int ret;
 	unsigned long collected;
 	LIST_HEAD(movable_page_list);
 
_
